### PR TITLE
22 feature dropdown menu for seeing sites or metrics

### PR DESCRIPTION
**Pull request by Grukank**
Bra att veta: Lintingen ogillar placeringen av import Box i App.tsx, men den behöver vara just där av någon anledning.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -4,6 +4,10 @@
 
 import React from 'react';
 import Menu from './components/Menu';
+import { GraphComponent, InfoboxComponent } from './components/BaseComponent';
+import Example from './components/Example';
+// Linting kommer klaga på att Box ligger här, men ligger den högre upp funkar inte webbsidan
```
Comment (AntonOstman) [inline src/App.tsx:9]: for some reason 
image
this function is undefined when box is imported before Example. Not sure if it is worth investigating further

**Review by rajtarn — CHANGES_REQUESTED** [review]
Looks good! Nice tests!

Some conflicts with main needs to be resolved, and the import problem will hopefully be resolved with that as well

File: src/App.tsx [hunk 2]
```diff
@@ -12,7 +16,27 @@ import './App.css';
...
+            display: 'flex',
+            backgroundColor: 'grey',
+          }}
+        >
+          <GraphComponent />
+          <InfoboxComponent />
+        </Box>
+      </Box>
+      <Example />
+      <p>Hejsan</p>
```
[the remark this answers is not in the document]
Comment (AlgotE) [reply src/App.tsx]: Conflicts with main has been resolver

File: src/App.css
Comment (rajtarn) [inline src/App.css:1]: Nice catch!

We could also call `.trim()` on the css variable to remove any padding.

**Review by CajsaWargren — APPROVED** [review]
it works, good job!

**Review by adsamu — APPROVED** [review]
looks good!

**Review by rajtarn — APPROVED** [review]
Good job!

**Pull request merged**